### PR TITLE
Minor doc update to class parallel::CellWeights.

### PR DESCRIPTION
**Pull request by bangerth**
Follow-up to #7305, part of #3511. @marcfehling: FYI.

### CODE REVIEW

File: include/deal.II/distributed/cell_weights.h [hunk 1]
```diff
@@ -32,11 +32,11 @@ namespace parallel
    * or by refinement/coarsening, cells will be distributed amongst all
    * subdomains to achieve an equally balanced workload. If the workload per
    * cell varies, which is in general the case for hp::DoFHandler objects, we
-   * can take those into account by introducing individual weights for
+   * can take that into account by introducing individual weights for
    * different cells.
    *
-   * This class allows to consult the FiniteElement, that it associated with
-   * each cell by the hp::DoFHandler, to determine the weight of the cell for
+   * This class allows to consult the FiniteElement that it associated with
```
Comment (jppelteret) [inline include/deal.II/distributed/cell_weights.h]: This class allows **one** to consult the FiniteElement that **is** associated with

Comment (bangerth) [reply include/deal.II/distributed/cell_weights.h]: I wrote this slightly differently, but you were on the right track. Thanks for reading it carefully!

**Comment by bangerth** [comment]
Updated version now here!

**Comment by jppelteret** [comment]
Since this is a docfix we can merge immediately.

**Pull request merged**